src/components/CardSingleSocial.vue

```diff
@@ -30,147 +30,4 @@
 
 <style scoped lang="scss">
 
-  .article-anchor-wrapper {
```

Comment:
why did all this go away? is it in _cards now? some of this scss is specific to how this component works, is it not?

Reply:
That's been accounted for.